jsapp/js/account/plan.component.tsx

```diff
@@ -0,0 +1,435 @@
+            featureItem.includes('feature_addon_')
+          ) {
+            expandBool = true;
+            break;
+          }
+        }
+      });
+    }
+    return expandBool;
+  };
+
+  const renderFeaturesList = (
+    items: Array<{
+      icon: 'positive' | 'positive_pro' | 'negative';
+      label: string;
+    }>,
+    title?: string
+  ) => {
+    return (
+      <>
+        <h2 className={styles.listTitle}>{title}</h2>
+        <ul>
+          {items.map((item) => (
+            <li key={item.label}>
+              <div className={styles.iconContainer}>
+                {item.icon !== 'negative' ? (
+                  <Icon
+                    name='check'
+                    size='m'
+                    classNames={
+                      item.icon === 'positive_pro'
+                        ? [styles.tealCheck]
+                        : [styles.stormCheck]
+                    }
+                  />
+                ) : (
+                  <Icon name='close' size='m' classNames={[styles.redClose]} />
+                )}
+              </div>
+              {item.label}
+            </li>
+          ))}
+        </ul>
+      </>
+    );
+  };
+
+  const returnListItem = (
+    type: string,
+    name: string,
+    featureTitle: string
+  ): ReactNode => {
+    let items: {
+      icon: 'positive' | 'positive_pro' | 'negative';
+      label: string;
+    }[] = [];
+    getListItem(type, name).map((listItem) => {
+      if (listItem.icon && name === 'Professional plan') {
+        items.push({icon: 'positive_pro', label: listItem.item});
+      } else if (!listItem.icon) {
+        items.push({icon: 'negative', label: listItem.item});
+      } else {
+        items.push({icon: 'positive', label: listItem.item});
+      }
+    });
+    return renderFeaturesList(items, featureTitle);
+  };
+
+  if (!state.products.length) {
+    return null;
+  }
+  return (
+    <div className={styles.accountPlan}>
+      <div className={styles.plansSection}>
+        <form className={styles.intervalToggle}>
+          <input
+            type='radio'
+            id='switch_left'
+            name='switchToggle'
+            value='year'
+            onChange={() => dispatch({type: 'year'})}
+            checked={!state.filterToggle}
+            aria-label={'Toggle to annual options'}
+          />
+          <label htmlFor='switch_left'>{t('Annual')}</label>
+
+          <input
+            type='radio'
+            id='switch_right'
+            name='switchToggle'
+            value='month'
+            aria-label={'Toggle to month options'}
+            onChange={() => dispatch({type: 'month'})}
+            checked={state.filterToggle}
+          />
+          <label htmlFor='switch_right'> {t('Monthly')}</label>
+        </form>
+
+        <div className={styles.allPlans}>
+          {filterPrices().map((price: Price) => (
+            <div className={styles.stripePlans} key={price.id}>
+              {isSubscribedProduct(price) ? (
```

Comment:
Yeah if there is nothing subscribed it should default to the community plan, this might be fixed in the updates coming from Jessica's branch. Do you know we could get it in this state to test?